targets/freertos.armv7m/freertos_drivers/cc32xx/sources

```diff
@@ -1,26 +0,0 @@
-include $(OPENMRNPATH)/etc/cc32xx.mk
-DEPS += TICC3200SDKPATH
-VPATH=$(OPENMRNPATH)/src/freertos_drivers/ti
-
-CXXSRCS += CC32xxUart.cxx \
-           CC32xxSPI.cxx \
-           CC32xxDeviceFile.cxx \
```

Comment:
hm, where did these go? I mean we still have these drivers so we should be compiling them I think?

Reply:
This build directory was for CC3200, I removed support for CC3200.